apps/prairielearn/src/lib/github.js

```diff
@@ -34,25 +34,19 @@ module.exports = {
     return new Octokit({ auth: config.githubClientToken });
   },
 
-  _waitAsync: function (millis) {
-    return new Promise((res, _rej) => {
-      setTimeout(res, millis);
-    });
-  },
-
   /**
    * Slugs a course shortname into a GitHub repository name.
-   * @param short_name Course shortname
+   * @param {string} short_name Course shortname
```

Comment:
Now that we've removed duplicate checking, I think we'll need to check whether this repo name already exists and append a `2` or something if it does?

Reply:
At the moment, I'm inclined to not do anything about this. For the general case of multiple institutions having the same course name, we should be embedding the institution name in the repo name, which should ensure uniqueness. I agree that this won't do anything for duplicates within an institution, but at the moment the creation job will fail immediately with a clear error message and the operator can try again with a unique name. How does that sound for now?

Reply:
Just how nice is the error message? A middle ground might be to explicitly check for a duplicate name within the institution and throw a very clear error in this case? "The course name 'CS 101' is already taken. Please try again with a different name."

Reply:
I was able to find an old job that failed for this reason, I think the error is pretty clear:

> Name already exists on this account

At any rate: at the moment, it'll only ever be Mariana or us handling these errors, so I'm not overly concerned about making this as clever as possible.

I'll land this in the current form. If it proves to be really confusing or error-prone in practice, we can easily add this on later.

Reply:
Great point that it's Mariana who will see this. I was thinking it was instructors!